scripts/docker-compose-mysql/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '2'
+services:
+  cromwell:
+    build: 
+      context: ./compose/cromwell
+    volumes:
+      - ./cromwell-executions:/cromwell-working-dir/cromwell-executions
+    command: ["/wait-for-it/wait-for-it.sh mysql-db:3306 -t 120 -- java -Dconfig.file=/app-config/application.conf -jar /app/cromwell.jar server"]
+    links:
+      - mysql-db
+    ports:
+      - "8800:8000"
```

Comment:
Allowing 80 is useful as it's way easier to use port 80 on GCS VMs if you can get away with it